application.go

```diff
@@ -95,6 +95,28 @@ type Fetch struct {
 	Cache      bool   `json:"cache"`
 }
 
+// GetAppOpts contains a payload for Application method
+//		id:		Filter the result to only return apps whose id is or contains the given value
+//		cmd:	Filter the result to only return apps whose cmd field contains the given value
+//		label:	A label selector query contains one or more label selectors, which are comma separated.
+//		embed:	Embeds nested resources that match the supplied path.
+// 				You can specify this parameter multiple times with different values
+type GetAppOpts struct {
+	Id    string   `url:"id,omitempty"`
```

Comment:
What's the purpose of this failed? Is it different from the URL parameter that Marathon often requires?

Also, being an [initialism](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) it should be named `ID` .